crates/bevy_ecs/src/storage/sparse_set.rs

```diff
@@ -50,7 +50,7 @@ macro_rules! impl_sparse_array {
             #[inline]
             pub fn contains(&self, index: I) -> bool {
                 let index = index.sparse_set_index();
-                self.values.get(index).map(|v| v.is_some()).unwrap_or(false)
+                self.values.get(index).and_then(Option::as_ref).is_some()
```

Comment:
This is equivalent as well, right? To me this is easier to read
```suggestion
                self.values.get(index).is_some_and(|v| v.is_some())
```

Reply:
I quite like how in the `and_then` version it flattens it down and ends on `.is_some()` rather than going "inside" the option but yeah that seems clearer.